bundles/org.openhab.core/src/main/java/org/openhab/core/util/ColorUtil.java

```diff
@@ -182,20 +183,17 @@ public static double[] hsbToXY(HSBType hsbType, Gamut gamut) {
         double[] xyY = new double[] { ((int) (q.x * 10000.0)) / 10000.0, ((int) (q.y * 10000.0)) / 10000.0,
                 ((int) (Y * 10000.0)) / 10000.0 };
 
-        LOGGER.trace("HSB: {} - RGB: {} - XYZ: {} {} {} - xyY: {}", hsbType, hsbType.toRGB(), X, Y, Z, xyY);
-
+        LOGGER.trace("HSB: {} - XYZ: {} {} {} - xyY: {}", hsbType, X, Y, Z, xyY);
```

Comment:
Why did you remove the intermediate RGB step from the logging?

Reply:
Because the method is deprecated

Reply:
`HSBType.toRGB` is deprecated, `ColorUtil.hsbToRgbPercent` is not. And please put that in a `if (LOGGER.isTraceEnabled()) { ... }` block, `hsbToRgbPercent` is quite complex and we don't need to calculate it all the time if we don't log it (that was already a problem before, it's not something new).

Reply:
Ok.